integration/service/connectivitymap/statistic.go

```diff
@@ -88,6 +88,11 @@ func (s *statistic) build(m ID2NearByIDsMap, MaxRange float64) *statistic {
 		s.MinOfMaxDistance = math.Min(s.MinOfMaxDistance, maxDistance)
 	}
 
+	if s.ValidCount == 0 {
+		glog.Warningf("connectivity's statistic detect 0 valid result.\n")
+		return s
+	}
```

Comment:
More information, please go to: https://github.com/Telenav/osrm-backend/issues/266

Also add protection in unit tests.